tests/dummy/app/app.js

```diff
@@ -4,7 +4,7 @@ import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
 // ember-browserify requires every package to be imported somewhere in the `app` tree
-import 'npm:common-tags';
+import 'common-tags';
```

Comment:
I've seen that we are using `yarn` here. I've updated the PR with a new `yarn.lock`. Let's see if it builds.